src/index.js

```diff
@@ -0,0 +1,43 @@
+import Store from '../modules/store';
+import Ui from '../modules/ui';
+import './style.css';
+
+class Task {
+  constructor(description, index) {
+    this.Description = description;
```

Comment:
- [ ] kindly ensure that a function for editing task descriptions is implemented. This will help if a wrong description is entered so it can be edited/corrected.